terraform/modules/quorum-cluster-region/bootnodes.tf

```diff
@@ -84,20 +93,36 @@ resource "aws_instance" "bootnode" {
 # ---------------------------------------------------------------------------------------------------------------------
 
 data "template_file" "user_data_bootnode" {
-  count = "${signum(lookup(var.bootnode_counts, var.aws_region, 0))}"
+
+  count = "${lookup(var.bootnode_counts, var.aws_region, 0)}"
 
   template = "${file("${path.module}/user-data/user-data-bootnode.sh")}"
 
   vars {
-    vault_dns  = "${var.vault_dns}"
-    vault_port = "${var.vault_port}"
+    constellation_s3_bucket  = "${aws_s3_bucket.quorum_constellation.id}"
+    index                    = "${count.index}"
+    bootnode_count_json      = "${data.template_file.bootnode_count_json.rendered}"
+    aws_region               = "${var.aws_region}"
+    primary_region           = "${var.primary_region}"
+    network_id               = "${var.network_id}"
+    use_elastic_bootnode_ips = "${var.use_elastic_bootnode_ips}"
+
+    # concat() is called to ensure there is always at least one element in the list,
+    # as element() cannot be called on empty list.  Solution is hacky, but lazy
+    # ternary evaluation will drop in Terraform 0.12: https://www.hashicorp.com/blog/terraform-0-1-2-preview
+    # If you're reading this and it has already released, try dropping the concat hack.
+    public_ip                = "${var.use_elastic_bootnode_ips ? element(concat(aws_eip.bootnodes.*.public_ip, list("")), count.index) : "nil"}"
```

Comment:
Whoops, should have been more specific about the convention. What I meant was to line up the user data vars. 

The convention is to line up `=` for any variables listed on consecutive lines. If you skip a line, the `=` don't need to align with the ones in the previous block.

So for example, `vault_dns` and `vault_port` should be aligned with each other, but don't need to align with `eip_id` above it or `consul_cluster_tag_key` below it